src/includes/class-doppler-for-woocommerce-app-connect.php

```diff
@@ -0,0 +1,212 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+/**
+ * This class handles the communication 
+ * with the api to connect or disconnect
+ * the app integration.
+ *  
+ * @since 1.0.2
+ * @author Hernán Galván <hgalvan@makingsense.com>
+ */
+class Doppler_For_WooCommerce_App_Connect {
+
+	const INTEGRATION = 'magento';
+	const DEBUG_MODE = true;
```

Comment:
This should remain `true`?

Reply:
Yes! (for now), before production will delete this constant, for now its just a constant for using a testing api key and account... gonna remove later.